### ISSUE TITLE
multipatch boundary id magic number

### ISSUE THREAD

**Issue by j042** [tataratat/splinepy#416]
BID=1 doesn't seem to have any effect, when you try to call
`mp.boundary_multipatch(1)`. is it because of default value -1?

**Comment by jzwar** [comment]
Yes. BID starts at `1`. Hence, `1` is default

**Comment by jzwar** [comment]
Yes. BID starts at `1`. Hence, `1` is default. Should you have more than one boundary, it should only return the first one, though.

**Comment by j042** [comment]
Ok, I guess it is helpful. Maybe we should add a note. I tried to set a certain boundary to BID=1, and I kept getting all the the boundaries. I will add this to upcoming clean up
